client/my-sites/domains/domain-management/settings/cards/name-servers-card.tsx

```diff
@@ -126,23 +138,34 @@ const NameServersCard = ( {
 				onClick={ handleLearnMoreClick }
 			/>
 		);
-		const notice = translate(
-			'By using custom name servers, you will manage your DNS records with your new provider, not WordPress.com. {{link}}Learn more{{/link}}',
-			{ components: { link } }
-		);
+
+		let notice;
+		if ( hasWpcomNameservers() ) {
+			notice = translate(
+				'Please do not set WordPress.com nameservers manually, toggle that on with the switch above. {{link}}Learn more{{/link}}',
```

Comment:
```suggestion
				'Please do not set WordPress.com name servers manually, toggle that on with the switch above. {{link}}Learn more{{/link}}',
```

Reply:
Small nit: we use both "nameserver" and "name server" interchangeably, but I _think_ the most correct one is "name server". Also, it seems it's the spelling we use the most in our support docs.

Reply:
Thanks and great timing! I'll make this update and freeze for translation 👍